app/src/main/java/io/pslab/activity/RoboticArmActivity.java

```diff
@@ -248,11 +257,14 @@ public void onStopTrackingTouch(SeekArc seekArc) {
         seekArc4.setOnSeekArcChangeListener(new SeekArc.OnSeekArcChangeListener() {
             @Override
             public void onProgressChanged(SeekArc seekArc, int i, boolean b) {
+                degreeText4.requestFocus();
                 if (editEnter) {
                     degreeText4.setText(String.valueOf(degree));
                     editEnter = false;
+                    degreeText4.setCursorVisible(false);
                 } else {
                     degreeText4.setText(String.valueOf((int) (i * 3.6)));
+                    degreeText4.setCursorVisible(false);
```

Comment:
Since this is in both if and else we can take it out and have only one statement